kernel/copy.masm

```diff
@@ -0,0 +1,256 @@
+# assert_eqw(A: Word, B: Word)
+#
+# Asserts that the top two words in the stack are equal
```

Comment:
For doc comments we should use `#!` rather than just `#` (this way, doc comments get included into the AST). 

Also, we usually describe the initial and the final state of the stack, and sometimes, for simple procedures like this, it might be useful to include a cycle count.

For this procedure, I'd probably have the following description:
```
#! Asserts that the top two words on the stack are equal.
#!
#! Stack transition look like so:
#! [A, B, ...] -> [...]
#!
#! This procedure requires 11 cycles.
```